### ISSUE TITLE
Feature Request Portal

### ISSUE THREAD

**Issue by dekkerglen** [dekkerglen/CubeCobra#1150]
As a user, I would like a set of pages where I can post bugs and feature requests, and other users can vote on which issues should be given the most attention. Users should also be able to comment on these posts.

**Comment by ruler501** [comment]
Couldn't github do this with reaction emojis as votes?

**Comment by dekkerglen** [comment]
Yes, but having an on site portal will drive more people to post - won't need to bring users off-site and need a github account

**Comment by dekkerglen** [comment]
Feature requests handled in discord now
